management/prompt.go

```diff
@@ -3,6 +3,9 @@ package management
 type Prompt struct {
 	// Which login experience to use. Can be `new` or `classic`.
 	UniversalLoginExperience string `json:"universal_login_experience,omitempty"`
+
+	// IdentifierFirst determines if the login screen prompts for just the identifier, identifier and password first.
+	IdentifierFirst bool `json:"identifier_first,omitempty"`
```

Comment:
Updated to `*bool`. I don't think the string field needs to be a pointer since an empty string will get handled by `omitempty` were `false` would not.

As a side note after running generate it looks like it generated a bunch of things from other PRs. It would be great to run everything up until the tests on every PR to ensure fmt/generate is good before merging to master.